### PR TITLE
Made description dynamic on related resources

### PR DESCRIPTION
**Pull request by GretaD**
### ☑️ Resolves

* ref https://github.com/nextcloud/contacts/pull/3670

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
BEFORE | Screenshot from 2023-10-31 14-52-33

### 🚧 Tasks

- [ ] Make descr dynamic
- [ ] Change the header styling to always be a primary colour

### 🏁 Checklist

- [ ] ⛑️ Tests are included or are not applicable
- [ ] 📘 Component documentation has been extended, updated or is not applicable

### CODE REVIEW

File: src/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.vue [hunk 1]
```diff
@@ -246,6 +249,7 @@ export default {
 .related-resources {
 	&__header {
 		margin: 0 0 10px 46px;
+		color: var(--color-primary-element);
```
Comment (Pytal) [inline src/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.vue:252]: We should not change the existing design, let's add an opt-in prop like https://github.com/nextcloud-libraries/nextcloud-vue/blob/c2095bfbaa9877a5920e7375c94f75390090e0ba/src/components/NcAppNavigationItem/NcInputConfirmCancel.vue#L81-L84 to conditionally make the header colored

File: src/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.vue [hunk 2]
```diff
@@ -120,6 +120,17 @@ export default {
...
+		description: {
+			type: String,
+			default: t('Anything shared with the same group of people will show up here'),
+		},
+		/**
+		 * If this element is used on a primary element set to true for primary styling.
+		 */
+		primary: {
+			default: false,
+			type: Boolean,
```
Comment (raimund-schluessler) [inline src/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.vue:132]:
```suggestion
			type: Boolean,
			default: false,
```

**Review by raimund-schluessler — APPROVED** [review]
Only a nitpick I have 😉

**Review by susnux — APPROVED** [review]
:+1: just squash your commits

**Comment by skjnldsv** [comment]
Please set appropriate `fix` or `enhancement` labels next time :+1:

**Pull request merged**